api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -68,7 +76,7 @@
 
     .button {
       height: 52px;
-      width: 178px;
+      width: 16rem;
```

Comment:
Nit: Could you also change the height to rem if we are making the width rem?

Reply:
i have already updated the username to a different password